awsgi/__init__.py

```diff
@@ -16,29 +17,49 @@ def convert_str(s):
         return s
 
 
-def response(app, event, context):
-    sr = StartResponse()
+def convert_b46(s):
+    return b64encode(s).decode("ascii")
```

Comment:
Following the existing style, could you use single quotes here and elsewhere? I realize that some linters prefer double quotes (and frankly I don't care which we use) but I think they should be consistent.

Reply:
Yes I apologise, I should have checked that. `black` auto formatted those lines. I've updated to single quotes :+1: